File: ueueo-framework/ueueo-domain-driven-desgin/src/main/java/com/ueueo/ddd/domain/entities/events/EntityChangeEntry.java
package com.ueueo.ddd.domain.entities.events;

import com.ueueo.auditing.EntityChangeType;
import lombok.Data;

/**
 * @author Lee
 * @date 2022-05-25 14:46
 */
@Data
public class EntityChangeEntry {
    private Object entity;
    private EntityChangeType changeType;

    public EntityChangeEntry(Object entity, EntityChangeType changeType) {
        this.entity = entity;
        this.changeType = changeType;
    }
}

File: ueueo-framework/ueueo-domain-driven-desgin/src/main/java/com/ueueo/ddd/domain/entities/events/EntityChangeEntry.cs
using System;
using Volo.Abp.Auditing;

namespace Volo.Abp.Domain.Entities.Events;

[Serializable]
public class EntityChangeEntry
{
    public Object Entity;// { get; set; }

    public EntityChangeType ChangeType;// { get; set; }

    public EntityChangeEntry(Object entity, EntityChangeType changeType)
    {
        Entity = entity;
        ChangeType = changeType;
    }
}
